bedrock/firefox/templates/firefox/all.html

```diff
@@ -13,7 +13,15 @@
   {% elif channel == 'esr' %}
     {{ _('Firefox ESR is intended for groups who deploy and maintain the desktop environment in large organizations. <a href="%s">Learn more.</a>')|format(url('firefox.organizations.organizations')) }}
   {% else %}
-    {{ _('Firefox is available in more than 80 languages thanks to the contributions of Mozilla community members from around the world.') }}
+    {{ _('Firefox is made in large part by volunteers around the world. That’s why we offer it in 80 languages (and why there’s heart and soul in every piece of code).') }}
```

Comment:
This is a new string not in an l10n tag, but I'm assuming this is ok and you are handling the `ja` locale?

The current one is still untranslated on prod since the last change: https://www.mozilla.org/ja/firefox/all/